src/messenger-shortcuts.js

```diff
@@ -127,7 +155,21 @@ function openHelp() {
   mute();
   setTimeout(function() {
     document.querySelector('div[role="dialog"] h2').innerHTML = "Keyboard Shortcuts for Messenger";
-    document.querySelector('div[role="dialog"] h2~div').innerHTML = HELP_HTML;
-    document.querySelector('div[role="dialog"] h2~div~div').remove();
-  }, 100);
+      document.querySelector('div[role="dialog"] h2~div').innerHTML = HELP_HTML;
+      document.querySelector('div[role="dialog"] h2~div~div').remove();
+    }, delay);    
+  delay = 0; // to prevent unnecessary timeout
```

Comment:
You set the timeout to 100ms, so every time you triggered the help dialog, you would be able to see the mute dialog for a brief moment. This way, you only get a delay the first time to make sure that it's loaded and after that, the delay is gone.